examples/no-repeated-options.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// How might I throw if an option is repeated?
+
+// 1. const { parseArgs } = require('node:util'); // from node
+// 2. const { parseArgs } = require('@pkgjs/parseargs'); // from package
+const { parseArgs } = require('..'); // in repo
+
+const options = {
+  ding: { type: 'boolean', short: 'd' },
+  beep: { type: 'boolean', short: 'b' }
+};
+const { values, tokens } = parseArgs({ options, tokens: true });
+
+// Loop over values and find the options that were repeated.
+const repeatedTokens = Object.keys(values)
+  // Make arrays of tokens for each used option name.
+  .map((name) => tokens.filter((t) => t.kind === 'option' && t.name === name))
+  .filter((used) => used.length > 1);
+if (repeatedTokens.length > 0) {
+  const optionsUsed = repeatedTokens[0].map((token) => token.rawName);
```

Comment:
```suggestion
  const optionsUsed = repeatedTokens.map((dups) => dups[0].rawName);
```
I think? If the goal is to list each of the options which occurs multiple times, anyway.

(If you only want to list the first thing which violates this rule, probably better to use `.find` rather than `.filter` on line 19.)

Reply:
I was aiming for all of the uses of the first thing which violates the rule. (But somewhat on a whim.)

So yes, find would be better than filter.

Reply:
I made it longer but simpler.